sh/tests/stress_test.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+echo "Test 4: Stress test with rapid commits"
+
+ORIGINAL_DIR=$(cd "$(dirname "$0")/.." && pwd)
+
+TEST_DIR=$(mktemp -d)
+cd "$TEST_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)